modules/mcuboot/Kconfig

```diff
@@ -112,4 +112,13 @@ config ADD_MCUBOOT_MEDIATE_SIM_FLASH_DTS
 	  multi image update of the network core or to allow serial recovery of
 	  the network core.
 
+config MCUBOOT_USE_ALL_AVAILABLE_RAM
+	bool "Allow MCUBoot to use all available RAM (security implications)"
+	depends on TRUSTED_EXECUTION_NONSECURE || TRUSTED_EXECUTION_SECURE
+	default y if (BOARD = thingy91_nrf9160)
```

Comment:
This was created targeting the non-secure thingy53 application, now that's been removed and replaced with the secure thingy91 core... What's going on here?

Reply:
Sorry pushed something that was going to be run on CI. I'll add both boards my bad.